addons/portal/static/src/scss/portal.scss

```diff
@@ -205,6 +205,9 @@ img, .media_iframe_video, .o_image {
 .oe_search_box {
     padding-right: 23px;
 }
+.oe_search_box::placeholder {
+    overflow: visible;
```

Comment:
Mmm, here it happens to fit but what if the text is even longer?

Might be better to set `text-overflow: ellipsis` on the search_box instead so we're sure not to introduce additional weirdness in here, if that seems ok to you

Reply:
Interesting, didn't know about that specific property yet. It's more appropriate than the one I used, but I'm going with [rdeodoo's](https://github.com/odoo/odoo/pull/135801#pullrequestreview-1635235635) suggestion instead of changing the placeholder string itself.